### ISSUE TITLE
[Designer] Bug 31588605: [Usable- Adaptive Cards - ExpenseReport.JSON]: Name of ER-13052 Link is not announced by the narrator by using Tab key.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7634]
### Problem Description

[Bug 31588605](https://microsoft.visualstudio.com/OS/_workitems/edit/31588605): [Usable- Adaptive Cards - ExpenseReport.JSON]: Name of ER-13052 Link is not announced by the narrator by using Tab key.\

User Experience:

Narrator users will be impacted if narrator will not announce the control name by using Tab key.

Test Environment:

Application: Adaptive Cards
OS version: 2004 (OS Build 21301.1000)

Application Version: 2101.28.1.0

Pre-Requisite: Launch the Adaptive Card application.

Repro steps:

1. Launch the Adaptive Card application and Enable Narrator.

2. Adaptive Card UWP Visualizer screen starts appearing. 

3. Navigate to ExpenseReport.Json button by using Tab key and activate it.

4. Navigate to ER-13052 link by using Tab key and verify the issue.

Actual Result:
Name of ER-13052 Link is not announced by the narrator by using Tab key. 

Observation:
Narrator announces the control name by using up and down arrow key in scan mode.

Note:
Working fine with NVDA Screen Reader.

Expected result:
Name of the control should be announced by the narrator while navigating by using Tab key.

### Card JSON

```JSON
Designer Bug. Repro on UWP Adaptive Card visualizer
```

**Comment by vagpt** [comment]
This issue is not repro on 1.0.2.0 as it now announcing as link ER-13052 hence closing this issue.

![Image](https://github.com/user-attachments/assets/729af947-91a0-48a7-8709-70211d320d78)
